docs/glossary.md

```diff
@@ -16,7 +16,7 @@ sidebar_label: Glossary
 - [**aragonUI**](/docs/aragonui-intro.html): Aragon-native toolkit of UI components for decentralized apps.
 - [**Aragon client**](http://app.aragon.org): Client to interact with aragonOS-powered decentralized organizations. It implements a signer with transaction pathing, notifications and a sandboxed environment for aragonOS-based apps using aragonAPI.
 - **DAO**: Stands for Decentralized Autonomous Organization. Here, a DAO means a set of aragonOS smart contracts.
-- [**DAO Kit**](/docs/kits-intro.html): Smart contracts that can be used as templates for creating pre-configured DAOs in just one transaction.
+- [**DAO Template**](/docs/templates-intro.html): Smart contracts that can be used as templates for creating pre-configured DAOs in just one transaction.
```

Comment:
Could we keep an entry for "DAO Kits" here that say they're deprecated for "DAO Templates"?